src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+    if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
```

Comment:
This condition is also too specific and does not scale. The logic should be generalized to calculate the cost based on the number of days rented.